programs/marginfi/src/instructions/marginfi_account/transfer_authority.rs

```diff
@@ -0,0 +1,50 @@
+use crate::{
+    events::{AccountEventHeader, MarginfiAccountTransferAccountAuthorityEvent},
+    prelude::*,
+    state::marginfi_account::MarginfiAccount,
+};
+use anchor_lang::prelude::*;
+
+pub fn set_account_transfer_authority(
+    ctx: Context<MarginfiAccountSetAccountAuthority>,
+) -> MarginfiResult {
+    // Gather accounts
+    let mut marginfi_account = ctx.accounts.marginfi_account.load_mut()?;
+    let new_account_authority = ctx.accounts.new_authority.load()?;
```

Comment:
No need to load this as if its an account loader, bc we are only interested in the address of the account info